boards/lilygo/ttgo_t8s3/ttgo_t8s3-pinctrl.dtsi

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Lothar Felten <lothar.felten@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32s3-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32s3-gpio-sigmap.h>
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_GPIO43>;
+			output-high;
+		};
+		group2 {
+			pinmux = <UART0_RX_GPIO44>;
+			bias-pull-up;
+		};
+	};
+
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <I2C0_SDA_GPIO1>,
+				 <I2C0_SCL_GPIO2>;
+			bias-pull-up;
+			drive-open-drain;
+			output-high;
+		};
+	};
+
+	spim2_default: spim2_default {
+		group1 {
+			pinmux = <SPIM2_MISO_GPIO13>,
+				 <SPIM2_SCLK_GPIO12>,
+				 <SPIM2_CSEL_GPIO10>;
+		};
+		group2 {
+			pinmux = <SPIM2_MOSI_GPIO11>;
+			output-low;
+		};
+	};
+
+	spim3_default: spim3_default {
+		group1 {
+			pinmux = <SPIM3_MISO_GPIO37>,
```

Comment:
The ESP32S3 module used in this board has SPIRAM, which is internally accessed with GPIOs 36/37/38. Sounds this pins should not be used for `spim3` as in here. I suggested to either remove ot update pins numbers.

Reply:
Hi @sylvioalves,
The PSRAM is connected to pins 28, 30, 31, 33, 34, 35
The Flash is connected to pins 30, 31, 32, 33, 34, 35
(One SPI controller with two CS fro PSRAM/Flash)
The spim3 uses pins 41, 42, 43, 44 (GPIO37, GPIO37, GPIO38, GPIO39)
I'm not sure if that matters, but the board uses separate ICs.
Is SPIM3 not available with this configuration?

Reply:
My bad, I forgot this board does not use an esp32s3 module. Sounds it is correct. 
I took a look now at the schematics, I could not find where GPIO36, GPIO37, GPIO38, GPIO39 are connected though. Is it only available as expansion pins?

Reply:
Hi Sylvio,
according to the schematic, GPIO36, GPIO37, GPIO38 and GPIO39 are only connected to the pin header for external usage.
![image](https://github.com/user-attachments/assets/7e12a3be-f721-42f9-977c-0c1f2092727b)
![image](https://github.com/user-attachments/assets/5c05b793-1300-4573-9af9-8634d8c8b2d0)